### PR TITLE
Clean up terminology and formatting. Edit for clarity.

### PR DESCRIPTION
**Pull request by austin-hilberg**

### CODE REVIEW

File: contents/queries/04-Operators.rst [hunk 1]
```diff
@@ -5,18 +5,20 @@
...
 Since the expressions are only symbolic representations, repeated ``fetch`` calls may yield different results as the state of the database is modified.
 
 DataJoint implements a complete algebra of operators on relations:
 O
 ===================================  =======================================   =================================================================================
 operator                             notation                                  meaning
 ===================================  =======================================   =================================================================================
-:ref:`restriction`                   ``A & cond``                              The subset of relation ``A`` the meets condition ``cond``
-:ref:`restriction`                   ``A - cond``  							  The subset of relation ``A`` the does not meet condition ``cond``
+:ref:`restriction`                   ``A & cond``                              The subset of table ``A`` that meets condition ``cond``
```
Comment (dimitri-yatsenko) [inline contents/queries/04-Operators.rst]: subset of entities

File: contents/queries/04-Operators.rst [hunk 2]
```diff
@@ -5,18 +5,20 @@
...
+:ref:`restriction`                   ``A - cond``  							  The subset of table ``A`` that does not meet condition ``cond``
```
Comment (dimitri-yatsenko) [inline contents/queries/04-Operators.rst]: subset of entities

File: contents/queries/04-Operators.rst [hunk 3]
```diff
@@ -5,18 +5,20 @@
 Operators
 =============
 
-[Data queries](Query basics) have the form of expressions using operators to derive the desired relation.  The expressions themselves do not contain any data.  They represent the desired data symbolically.
+:ref:`Data queries <queries>` have the form of expressions using operators to derive the desired table.
+The expressions themselves do not contain any data.
+They represent the desired data symbolically.
 
-Once a query is formed, the [`fetch`](Fetching) methods are used to bring the data into the MATLAB or Python workspace.
+Once a query is formed, the :ref:`fetch <fetc>` methods are used to bring the data into the local workspace.
```
Comment (dimitri-yatsenko) [inline contents/queries/04-Operators.rst]: broken link

**Review by dimitri-yatsenko — CHANGES_REQUESTED** [review]

File: contents/queries/04-Operators.rst (hunk 1 above)
Comment (austin-hilberg) [reply contents/queries/04-Operators.rst]: `subset of entities in table A` ?

Comment (dimitri-yatsenko) [reply contents/queries/04-Operators.rst]: from table A

**Review by dimitri-yatsenko — APPROVED** [review]

**Pull request merged**